.github/workflows/CI.yml

```diff
@@ -53,6 +53,13 @@ jobs:
         with:
           toolchain: ${{ matrix.rust }}
           override: true
+          
+      - run: |
+          # install blueprint-compiler
+          git clone --depth 1 https://gitlab.gnome.org/jwestman/blueprint-compiler
+          cd blueprint-compiler
+          meson _build
+          sudo ninja -C _build install
```

Comment:
Should be submitted as a separate PR and change the Dockerfile instead.

Reply:
I got that error with that CI file, but I can't get it locally

![Screenshot from 2023-01-09 17-01-52](https://user-images.githubusercontent.com/63008755/211325787-fd58a7d6-7ee7-420a-8e4f-2d5599acc870.png)

Reply:
#1239